### DISCUSSION TITLE
edgetunnel.anycast.eu.org 废弃了吗

### DISCUSSION THREAD

**Discussion by Bitcion** [XIU2/CloudflareSpeedTest#487] [讨论]
### 问题描述

之前把这个ip添加到workers节点可以访问gpt，现在几个相关的反带ip域名只有这个废弃了，其他几个能用但是无法使用chatgpt。
解析了了发现是日本甲骨文的ip，难道是日本甲骨文ip没有被chatgpt禁用吗，或者可以再提供几个日本甲骨文的反带ip测试一下

### 软件版本

 v2.2.2

**Comment by XIU2** [comment]
和本项目无关的不要提 Issues，给你转成 discussions 了。
